test/unit/x20schemapack.test.js

```diff
@@ -62,6 +62,26 @@ describe('Convert method', function() {
     });
     done();
   });
+
+  it('Should convert an example file from: rangeMediaType.json', function(done) {
+    const fileSource = path.join(__dirname, SWAGGER_20_FOLDER_JSON, 'rangeMediaType.json'),
+      fileData = fs.readFileSync(fileSource, 'utf8'),
+      input = {
+        type: 'string',
+        data: fileData
+      },
+      schemapack = new SchemaPack(input);
+
+    schemapack.convert((error, result) => {
+      expect(error).to.be.null;
+      expect(result.output[0].data.item[0].response[0]._postman_previewlanguage).to.equal('json');
+      expect(result.output[0].data.item[0].response[0].body).to.not.be.empty;
+      expect(result.output[0].data.item[0].response[0].header.find((header) => {
+        return header.key === 'Content-Type';
+      }).value).to.equal('*/*');
+    });
+    done();
```

Comment:
@LuisTejedaS `done();` should be called inside callback when all checks pass. This causes execution problems, as callback that marks the finish of the test case is called before what the test aims to achieve.